File: posts/php/if-else.php
<?php
/*
Condicionais no PHP
if, elseif, else e switch
*/
$idade = 18;

if ($idade >= 18) {
    echo "Maior de idade\n";
} else {
    echo "Menor de idade\n";
}

$nota = 85;
if ($nota >= 90) {
    echo "Conceito A";
} elseif ($nota >= 80) {
    echo "Conceito B";
} else {
    echo "Conceito C";
}

File: posts/go/if-else.go
/*
Condicionais em Go

if/else sem parênteses no bool
*/
package main

import "fmt"

func main() {
	idade := 18

	if idade >= 18 {
		fmt.Println("Maior de idade")
	} else {
		fmt.Println("Menor de idade")
	}

	nota := 85
	if nota >= 90 {
		fmt.Println("A")
	} else if nota >= 80 {
		fmt.Println("B")
	} else {
		fmt.Println("C")
	}
}
